lib/exposes.js

```diff
@@ -241,6 +247,11 @@ class Light extends Base {
         }
         this.features.push(new Numeric('color_temp_startup', access.ALL).withUnit('mired').withValueMin(range[0]).withValueMax(range[1])
             .withDescription('Color temperature after cold power on of this light')
+            .withPreset('coolest', range[0], 'Coolest temperature supported')
+            .withPreset('cool', 250, 'Cool temperature (250 mireds)')
```

Comment:
Not sure about mireds for description, entire my life I used Kelvins to describe light temperature

Reply:
Like on any light bulb box you'll see Kelvin not mired

Reply:
color_temp uses mireds though, and it's the exact value we pass.

Reply:
Yes, but in the title we can pass kelvin's 
Cool temperature (xxxxK)

Reply:
I find it kind of confusing if we list Kelvin here but the attribute required Mireds, but having both Kelvin and Mireds in the description seems good.